### ISSUE TITLE
Mac M1 chip agbcc build error

### ISSUE THREAD

**Issue by tkolarik** [jiangzhengwenjz/katam#149]
This was an error on the main branch of agbcc as well, though i believe it was fixed there.

`ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

make: *** [cc1] Error 1

mv: rename gcc_arm/cc1 to agbcc_arm: No such file or directory`

**Comment by jiangzhengwenjz** [comment]
As https://github.com/pret/agbcc/pull/52 has been merged, is this resolved?

**Comment by jiangzhengwenjz** [comment]
No response in 3 years.

**Comment by tkolarik** [comment]
Yeah never got a chance to test it but if no one else reports issues its probably safe to consider it resolved
